presto-postgresql/src/test/java/io/prestosql/plugin/postgresql/TestPostgreSqlTypeMapping.java

```diff
@@ -268,6 +269,23 @@ public void testTimestamp()
         // testing this is hard because of https://github.com/prestodb/presto/issues/7122
     }
 
+    @Test
+    public void testJson()
```

Comment:
in this test, also add an INSERT via Presto to these json, jsonb columns

Reply:
done